pkgs/by-name/ba/basalt-monado/package.nix

```
@@ -0,0 +1,112 @@
+{ stdenv,
+  lib,
+  boost,
+  eigen,
+  tbb,
+  fmt,
+  bzip2,
+  opengv,
+  cli11,
+  magic-enum,
+  sophus,
+  cereal,
+  freeglut,
+  glew,
+  libGL,
+  opencv,
+  pangolin,
+  fetchFromGitLab,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  autoPatchelfHook,
+  extra-cmake-modules,
+  nix-update-script
+}:
```

This list is out of order.
```suggestion
{ stdenv,
  lib,
  fetchFromGitLab,
  fetchFromGitHub,
  autoPatchelfHook,
  boost,
  bzip2,
  cereal,
  cli11,
  cmake,
  eigen,
  extra-cmake-modules,
  fmt,
  freeglut,
  glew,
  libGL,
  magic-enum,
  opencv,
  opengv,
  pangolin,
  pkg-config,
  sophus,
  tbb,
  nix-update-script
}:
```

---

Also fixed now.